skunkworks_crow/src/main/java/org/odk/share/bluetooth/BluetoothUtils.java

```diff
@@ -0,0 +1,76 @@
+package org.odk.share.bluetooth;
+
+import android.app.Activity;
+import android.bluetooth.BluetoothAdapter;
+import android.os.Build;
+
+import java.util.UUID;
+
+/**
+ * Basic tools for bluetooth features.
+ *
+ * @author huangyz0918 (huangyz0918@gmail.com)
+ * @since 04/06/2019
+ */
+public class BluetoothUtils {
+
+    // UUID for security bluetooth pairing.
+    public static final UUID SPP_UUID = UUID.fromString("724f00a0-795e-4272-a85a-11075e760e58");
```

Comment:
Is this required to be constant? Or it can be generated randomly at runtime.
It would be good if generate it randomly at runtime.

Reply:
Improved.